style.css

```diff
@@ -0,0 +1,13 @@
+.first {
+  margin: auto;
+  width: 50%;
+  border: 3px solid green;
+  padding: 10px;
+}
+
+h1 {
+  color: rgba(13, 13, 117, 0.623);
+  margin: auto;
+  width: 50%;
```

Comment:
You need to use `display:inline-block` if you wish to specify the width of h1 tag